src/Avalonia.Base/Input/GestureRecognizers/VelocityTracker.cs

```diff
@@ -180,7 +180,7 @@ public void AddPosition(TimeSpan time, Vector position)
         internal Velocity GetVelocity()
         {
             var estimate = GetVelocityEstimate();
-            if (estimate == null || estimate.PixelsPerSecond.IsDefault)
+            if (estimate == null || estimate.PixelsPerSecond == default(Vector))
```

Comment:
The type parameter in `default(T)` is used whenever type is unclear. This is about the only case it wasn't clear though and most other code just uses `default`.